### DISCUSSION TITLE
Bandolier and Safety harness go in toolbelt slot.

### DISCUSSION THREAD

**Discussion by 8on8er** [FakeFishGames/Barotrauma#11947] [Ideas and suggestions]
My suggestion is the bandolier and safety harness no longer occupy the body slot but instead the tool belt slot.

- I also think both items should require a toolbelt in the recipe.
- Safety harness should give +5 to electrical and mechanics.
- safety harness gets 4 inventory slots
- safety harness only gives stun resistance when stun is inflicted by junction box/mechanism

This would also resolve the issue where bots unequip the bandolier or safety harness when the sub floods.

**Comment by MarkCringan** [comment]
The primary utility for the safety harness in my opinion is the resistance to the force of rushing water.  Putting it in toolbelt slot would allow you to stack diving suits and safety harness, which is too powerful in my opinion. They should be tradeoffs, NOT stackable.

**Reply by Randy624** [reply]
Use bandolier mod.
https://steamcommunity.com/sharedfiles/filedetails/?id=2921805896
